.eslintrc

```diff
@@ -44,13 +50,9 @@
   },
   "overrides": [
     {
-      "files": ["**/*.js", "**/*.jsx", "**/*.ts"],
+      "files": ["*.json"],
```

Comment:
Flipped the logic here. Instead of adding extensions one by one, mentioned the one we do not want to have headers (it is only JSON)